tests/Composer/Installers/Test/InstallerTest.php

```diff
@@ -270,6 +272,8 @@ public function dataForTestInstallPath()
             array('puppet-module', 'modules/puppet-name/', 'puppet/puppet-name'),
             array('redaxo-addon', 'redaxo/include/addons/my_plugin/', 'shama/my_plugin'),
             array('redaxo-bestyle-plugin', 'redaxo/include/addons/be_style/plugins/my_plugin/', 'shama/my_plugin'),
+            array('reindex-theme', 'themes/my_module/', 'author/my_module'),
+            array('reindex-plugin', 'plugin/my_module/', 'author/my_module'),
```

Comment:
Yep, it's plugins actually, my mistake.

array('reindex-theme', 'themes/my_module/', 'author/my_module'),
array('reindex-plugin', 'plugins/my_module/', 'author/my_module'),